File: web.py
from flask import Flask
app = Flask(__name__)

@app.route("/")
def hello():
    print("hello world!")
    return "\nworks!\n"

if __name__ == "__main__":
    app.run(host = '0.0.0.0', ssl_context = ("server.pem", "cert.key"))

File: web.go
package main

import (
	"log"
	"net/http"
)

func helloServer(w http.ResponseWriter, req *http.Request) {
	w.Header().Set("Content-Type", "text/plain")
	w.Write([]byte("This is an example server.\n"))
}

func main() {
	http.HandleFunc("/hello", helloServer)
	err := http.ListenAndServeTLS(":5000", "server.pem", "cert.key", nil)
	if err != nil {
		log.Fatal("ListenAndServe: ", err)
	}
}
